discovery-infra/tests/conftest.py

```diff
@@ -24,7 +24,9 @@
 env_variables = {"ssh_public_key": utils.get_env('SSH_PUB_KEY'),
                  "remote_service_url": utils.get_env('REMOTE_SERVICE_URL'),
                  "pull_secret": utils.get_env('PULL_SECRET'),
+                 "second_pull_secret": utils.get_env('SECOND_PULL_SECRET'),
```

Comment:
why os env? it is not something test specific?

Reply:
i am not sure somebody will ever remember to add SECOND_PULL_SECRET to run it's test same with SECOND_OFFLINE_TOKEN